src/test/java/posters/tests/unit/LoginTest.java

```diff
@@ -44,9 +40,9 @@ public void setup()
     @DataSet(1)
     public void testSuccessfulLogin()
     {
-        LoginPage loginPage = prepareTest();
+        var loginPage = prepareTest();
```

Comment:
move this into a `@Before` method of the class if this should be executed automatically for all methods